pyodide_build/cli/config.py

```diff
@@ -33,7 +34,8 @@ def _get_configs() -> dict[str, str]:
 
     configs: dict[str, str] = get_build_environment_vars(get_pyodide_root())
 
-    configs_filtered = {k: configs[v] for k, v in PYODIDE_CONFIGS.items()}
+    configs_filtered = {k: configs.get(v, "") for k, v in PYODIDE_CONFIGS.items()}
```

Comment:
I think PYODIDE_CONFIGS should be strictly a subset of the return value of `get_build_environment_vars`. Showing the empty value that does not exist would not be very useful.

Reply:
Yes, that is fair. I dropped this change in 8280f05956533c7200fda6090431d7f3e463c124